packages/suite/src/components/suite/TransactionsGraph/index.tsx

```diff
@@ -193,7 +193,7 @@ const TransactionsGraph = React.memo((props: Props) => {
                                             receivedValueFn={props.receivedValueFn}
                                             balanceValueFn={props.balanceValueFn}
                                             extendedDataForInterval={extendedDataForInterval}
-                                            onShow={index => setHovered(index)}
+                                            onShow={(index: HoveredIndex) => setHovered(index)}
```

Comment:
why do we even use this custom typing `HoveredIndex`? shouldn't it be just a `number` ?

Reply:
Originally I thought I'd use `undefined` as a part of the type, but then I switched to `-1` for the default state so the `number` type is enough. Fixed 👍